src/locale/locales/fr/messages.po

```diff
@@ -351,11 +321,11 @@ msgstr "Vous confirmez ? Cela ne pourra pas être annulé."
 
 #: src/view/com/composer/select-language/SuggestedLanguage.tsx:60
 msgid "Are you writing in <0>{0}</0>?"
-msgstr ""
+msgstr "Ecrivez-vous en <0>{0}</0> ?"
```

Comment:
Accents should be added on top of capital letters too. (Added a non-breaking space before the question mark, too.)

```suggestion
msgstr "Écrivez-vous en <0>{0}</0> ?"
```